testdata/manifest/golden.toml

```diff
@@ -2,13 +2,13 @@ ignored = ["github.com/foo/bar"]
 
 [[constraint]]
```

Comment:
This whole commit is a WIP. I wanted to demonstrate how the libraries would be used. Since the output of the libraries would be visible only in failed tests, I had to fail these two tests on purpose. However, this will of course not go into final PR.